src/bin/lockin-external.rs

```diff
@@ -142,187 +121,26 @@ const APP: () = {
             .last()
             .unwrap();
 
-        // convert i/q to power/phase,
-        let power_phase = true; // TODO: expose
-
-        let mut output = if power_phase {
+        let conf = "frequency_discriminator";
```

Comment:
I agree. In this case it was a bit more code. And since the enum would be moved into the settings structure and exposed via miniconf, it seems ok to improve when doing that refactor.